crypto/asn1/a_time.c

```diff
@@ -124,14 +124,14 @@ int asn1_time_to_tm(struct tm *tm, const ASN1_TIME *d)
             i++;
             break;
         }
-        if (!isdigit(a[o]))
+        if (!ossl_isdigit(a[o]))
             goto err;
         n = a[o] - '0';
```

Comment:
Likewise, the behaviour is unchanged with the new code, so I'll leave this as is.